<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddIngredientDialog</class>
 <widget class="QDialog" name="AddIngredientDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1401</width>
    <height>865</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>200</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="priceLineEdit">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>270</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="stockLineEdit">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>340</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="supplierLineEdit">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>410</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="submitButtom">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>490</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
